public/wp-content/themes/glific/front-page.php

```diff
@@ -110,6 +110,95 @@
+					</div>
+					<a class="carousel-control-prev" href="#user_satying" role="button" data-slide="prev">
+						<span class="carousel-control-prev-icon" aria-hidden="true"></span>
+						<span class="sr-only">Previous</span>
+					</a>
+					<a class="carousel-control-next" href="#user_satying" role="button" data-slide="next">
+						<span class="carousel-control-next-icon" aria-hidden="true"></span>
+						<span class="sr-only">Next</span>
+					</a>
+				</div>
+			<?php endif; ?>
+		</div>
+	<?php endif ; ?>
+
+	<?php $know_more = get_field('know_more');
+		if (!empty($know_more)) : ?>
+		<div class="d-flex flex-column w-full mx-auto bg-theme-gray pb-26">
+			<div class="d-flex flex-row bg-white box-shadow-dark-10 mt-10 w-344 w-md-full p-6 ml-auto mr-md-0 justify-content-center align-items-md-center align-items-start d-xl-none rounded-top-left-30 rounded-bottom-left-30 rounded-md-0 rounded-xl-top-right-30">
+				<h5 class="w-150 w-md-auto mr-md-21 font-heebo-medium fz-24 leading-35 mb-6"><?php echo $know_more['whatsapp']['text'] ; ?></h5>
+				<a class="glific-button-border bg-theme-secondary w-156 py-4 w-md-170 fz-18 leading-27 text-center text-white font-heebo-bold mt-6 text-decoration-none mt-md-0" href="<?php echo $know_more['whatsapp']['button']['url'] ; ?>"><?php echo $know_more['whatsapp']['button']['link'] ; ?></a>
+			</div>
+			<div class="d-flex flex-column flex-md-row mt-md-10">
+				<div class="w-md-half mr-md-6 d-xl-flex flex-xl-column">
+					<div class="d-xl-flex flex-column  bg-white box-shadow-dark-10 mt-10 w-344 w-md-full p-6 mr-auto mr-md-0 justify-content-center align-items-center d-none w-xl-full ml-xl-auto mt-xl-0 p-xl-10 rounded-xl-top-right-30">
+						<div class="w-xl-432 ml-xl-auto">
+							<h5 class="w-150 w-md-auto font-heebo-medium fz-24 leading-35 mb-6"><?php echo $know_more['whatsapp']['text'] ; ?></h5>
+							<a class="glific-button-border bg-theme-secondary w-156 py-4 w-md-170 fz-18 leading-27 text-center text-white font-heebo-bold mt-6 text-decoration-none mt-md-0 d-block" href="<?php echo $know_more['whatsapp']['button']['url'] ; ?>"><?php echo $know_more['whatsapp']['button']['link'] ; ?></a>
+						</div>
+					</div>
+					<div class="bg-white d-flex flex-column box-shadow-dark-10 mt-10 mt-md-0 w-344 w-md-full p-6 mr-auto ml-md-0 h-md-full w-xl-full ml-xl-auto mt-xl-10 p-xl-10 rounded-top-right-30 rounded-bottom-right-30">
+						<div class="w-xl-432 ml-xl-auto">
+							<h5 class="font-heebo-medium fz-24 leading-35 mb-6"><?php echo $know_more['open_source']['text'] ; ?></h5>
+							<p class="mb-0 font-heebo-regular fz-18 leading-27 mb-6"><?php echo $know_more['open_source']['content'] ; ?></p>
+							<a class="glific-button-border bg-theme-primary py-4 w-170 fz-18 leading-27 text-center text-white font-heebo-bold mt-6 text-decoration-none mt-md-0 d-block" href="<?php echo $know_more['open_source']['button']['url'] ; ?>"><?php echo $know_more['open_source']['button']['link'] ; ?></a>
```

Comment:
Please try breaking out the lenghier lines of code in multiple lines for more readability.